package/plugin-gargoyle-stamgr/Makefile

```diff
@@ -0,0 +1,69 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=plugin_gargoyle_stamgr
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/plugin-gargoyle-stamgr
+	SECTION:=admin
+	CATEGORY:=Administration
+	SUBMENU:=Gargoyle Web Interface
+	TITLE:=STA Manager for Gargoyle
+	MAINTAINER:=Michael Gray <support@lantisproject.com>
+	DEPENDS:=+gargoyle iwinfo
```

Comment:
+iwinfo

?

Reply:
Only want it to rely on it, not select it. This should stop it being available on non wireless targets.

Reply:
Added additional dependencies i'd missed and switched this to the select format. It shouldn't do any harm and is more consistent with the other plugins.